model/Uzivatel/Finance.php

```diff
@@ -453,92 +302,125 @@ public function prehledPopis(): string
         foreach ($this->serazenyPrehled() as $r) {
             $out[] = $r['nazev'] . ' ' . $r['castka'];
         }
+
         return implode(', ', $out);
     }
 
+    private function prehled(): array
+    {
+        if ($this->prehled === null) {
+            if ($this->logovat) {
+                $this->prepocti();
+            }
+        }
+
+        return $this->prehled ?? [];
+    }
+
```

Comment:
Tady mi to dává větší smysl vyřešit takhle. Když už jednou dostanu zpátky nějakou hodnotu tak by měla být už pak uložená ve fieldu.
```suggestion
    private function prehled(): array
    {
        if ($this->prehled === null) {
            if ($this->logovat) {
                $this->prepocti();
            }
            $this->prehled ??= [];
        }

        return $this->prehled;
    }

```

Reply:
Je to tak, upravil jsem